app.js

```diff
@@ -89,7 +89,7 @@ const styles = StyleSheet.create({
     backgroundColor: Platform.OS === 'ios' ? c.iosLightBackground : c.androidLightBackground,
   },
   navigationBar: {
-    backgroundColor: c.olevilleGold,
+    backgroundColor: c.tint,
```

Comment:
Wow I'm blind. Thanks @drewvolz. I think I'd like to keep in the same color family as `olevilleGold`, but if we want to go with a darker variation that's ok. Maybe one of these:
* `#C28F00`
* `#987000`

@hawkrives what do you think of those?